storagenode/piecestore/endpoint.go

```diff
@@ -643,7 +645,8 @@ func (endpoint *Endpoint) Download(stream pb.DRPCPiecestore_DownloadStream) (err
 			mon.IntVal("download_failure_duration_ns", actionSeriesTag).Observe(downloadDuration)
 			mon.FloatVal("download_failure_rate_bytes_per_sec", actionSeriesTag).Observe(downloadRate)
 			if errors.Is(err, context.Canceled) {
-				endpoint.log.Info("download failed", zap.Stringer("Piece ID", limit.PieceId), zap.Stringer("Satellite ID", limit.SatelliteId), zap.Stringer("Action", limit.Action), zap.Int64("Offset", chunk.Offset), zap.Int64("Size", downloadSize), zap.String("Remote Address", remoteAddr), zap.Error(err))
+				endpoint.log.Info("download canceled (context canceled)", zap.Stringer("Piece ID", limit.PieceId), zap.String("Reason", "Someone else downloaded the required piece faster."))
+				endpoint.log.Debug("download canceled (context canceled)", zap.Stringer("Piece ID", limit.PieceId), zap.Stringer("Satellite ID", limit.SatelliteId), zap.Stringer("Action", limit.Action), zap.Int64("Offset", chunk.Offset), zap.Int64("Size", downloadSize), zap.String("Remote Address", remoteAddr), zap.Error(err))
```

Comment:
We probably don't need the "` (context canceled)`" hint on this line, since the err is still being provided.